### DISCUSSION TITLE
Profile description / bio

### DISCUSSION THREAD

**Discussion by MrKrisKrisu** [Traewelling/traewelling#2005] [Ideas / Feature Requests]
**Describe the solution you'd like**
As a user I would like to introduce myself in a profile description. Similar to Twitter or Mastodon.

**Additional context**
Emoji support, limited characters, ...

**Comment by yannickkirschen** [comment]
Hi, are there any plans to implement this feature? This would be really cool 😍

**Comment by sanojkl** [comment]
is there, so i think this can be marked as done

**Reply by yannickkirschen** [reply]
Yes, that's true. I saw the new feature a couple of days ago and even forgot that I opened this thread some time ago 😂. Thanks for implementing 🙂

**Comment by marhei** [comment]
Implemented with #3495
